### PR TITLE
Bootstrap + new CSS

### PR DESCRIPTION
**Pull request by nicr9**
...

### CODE REVIEW

File: kanji/kanji.py [hunk 1]
```diff
@@ -16,14 +16,16 @@
...
     with open(KANJI_PATH, 'r', encoding="utf8") as inp:
         reader = csv.reader(inp, delimiter=',')
         all_kanji = [
                 x for x in reader
                 ]
-    return render_template("homepage.html", kanji=random.sample(all_kanji, 4))
+    return render_template("Kanji_Flash_Cards.html",kanji=random.sample(all_kanji, 4))
 
 @app.route("/reload")
+@app.route("/reload")
```
Comment (nicr9) [inline kanji/kanji.py:28]: Please delete this line

File: kanji/startbootstrap_agency_1.0.6/LICENSE [hunk 2]
```diff
@@ -0,0 +1,202 @@
+Apache License
```
Comment (nicr9) [inline kanji/startbootstrap_agency_1.0.6/LICENSE:1]: Could you delete the startbootstrap_agency_1.0.6/ folder too?

File: kanji/templates/index2.html [hunk 3]
```diff
@@ -0,0 +1,583 @@
+<!DOCTYPE html>
```
Comment (nicr9) [inline kanji/templates/index2.html:1]: This file isn't needed anymore

**Comment by nicr9** [comment]
Please make "pronunciation" and "english" columns white text

**Comment by nicr9** [comment]
Actually, use roll over instead of white text!

File: kanji/static/style.css [hunk 4]
```diff
@@ -1,4 +1,4 @@
-body            { font-family: sans-serif; background: #eee; }
+body            { font-family: sans-serif; background: #0b76ba; }
 a, h1, h2       { color: #377ba8; }
 h1, h2          { font-family: 'Georgia', serif; margin: 0; }
 h1              { border-bottom: 2px solid #eee; }
```
Comment (nicr9) [inline kanji/static/style.css:4]: Remove `border-bottom`

File: kanji/kanji.py (hunk 1 above)
Comment (professorc0ak) [inline kanji/kanji.py:28]: done

File: kanji/startbootstrap_agency_1.0.6/LICENSE (hunk 2 above)
Comment (professorc0ak) [inline kanji/startbootstrap_agency_1.0.6/LICENSE:1]: done

File: kanji/templates/index2.html (hunk 3 above)
Comment (professorc0ak) [inline kanji/templates/index2.html:1]: deleted

**Comment by nicr9** [comment]
This PR is superseded by #2. Closing

**Pull request closed without merging**